File: libs/sdk-java/src/main/java/io/daytona/sdk/model/Command.java
// Copyright Daytona Platforms Inc.
// SPDX-License-Identifier: Apache-2.0

package io.daytona.sdk.model;

public class Command extends io.daytona.toolbox.client.model.Command {
    public Command() {}

    public Command(io.daytona.toolbox.client.model.Command source) {
        super();
        if (source != null) {
            setId(source.getId());
            setCommand(source.getCommand());
            setExitCode(source.getExitCode());
        }
    }
}

File: apps/daemon/pkg/toolbox/process/coderun/command.go
// Copyright 2025 Daytona Platforms Inc.
// SPDX-License-Identifier: AGPL-3.0

package coderun

import "strings"

func formatArgv(argv []string) string {
	if len(argv) == 0 {
		return ""
	}

	return strings.Join(argv, " ")
}
